README.md

```diff
@@ -165,6 +166,21 @@ This covers the following environment variables:
 
 For information on what these mean and how to configure them, please see [this wiki page](https://wiki.folio.org/display/FOLIOtips/Queue+and+chunk+processing+customization).
 
+## System user
+
+To enable asynchronous job launching (as part of the file splitting process), the module creates
+a system user upon installation.  By default, the system user is named `data-import-system-user`,
```

Comment:
The standard for other modules is to have a system user specific to the module. The name of the system user points to Data Import in general not mod-data-import. So I think we should follow the existing scheme for system users so that we know which system user belongs to which module.

Reply:
What would you suggest?  Some copy the module names, while others (data export, pubsub, etc) don't:

<img width="768" alt="image" src="https://github.com/folio-org/mod-data-import/assets/8005215/9f26abd0-a9f2-47f1-90ec-96c133e87929">

Reply:
"mod-data-import", otherwise you can leave it as is since there is no consistency with other modules.